resources/CMakeLists.txt

```diff
@@ -19,12 +19,12 @@ set(RESOURCES_FILES
 )
 
 foreach(resource ${RESOURCES_FILES})
-  configure_file(${resource} Resources/${resource} COPYONLY)
+  configure_file(${resource} ${resource} COPYONLY)
 endforeach()
 
-add_executable(Resources IMPORTED GLOBAL)
+add_library(Resources INTERFACE IMPORTED GLOBAL)
 set_target_properties(Resources
   PROPERTIES
-    IMPORTED_LOCATION ${CMAKE_CURRENT_BINARY_DIR}/Resources/IF_YOU_SEE_THIS_YOU_TRIED_TO_RUN_THE_RESOURCES_TARGET
+    RESOURCE "${RESOURCES_FILES}"
```

Comment:
A neat thing about `RESOURCE` is that if you don't have any "linkable" outputs, you don't need to specify one. Will this break future consumers? Probably. Do I care right now? Nope.